dot/node.go

```diff
@@ -147,6 +148,11 @@ func NewNode(cfg *Config, ks *keystore.Keystore) (*Node, error) {
 
 	var nodeSrvcs []services.Service
 
+	// Message Channels (send and receive messages between services)
+
+	coreMsgs := make(chan network.Message)
+	networkMsgs := make(chan network.Message)
```

Comment:
could you add a comment explaining these? like which one sends to network, which one sends to core, etc

Reply:
coreMsgs are messages sent from core service to network service
networkMsgs are message sent from network service to core service

will add :+1: